.github/workflows/load-test.yaml

```diff
@@ -132,7 +132,7 @@ jobs:
                   PLAYBACK_MANIFEST_URL="https://${{ inputs.region }}-staging-staging-catalyst-${{ inputs.pod-index }}.livepeer.monster/webrtc/video+%s"
                   if [ "${{ inputs.playback-protocol }}" = "HLS" ]; then
                       PLAYBACK_BASE_URL="${PLAYBACK_BASE_URL}?lowLatency=false"
-                      PLAYBACK_MANIFEST_URL="https://${{ inputs.region }}-prod-catalyst-${{ inputs.pod-index }}.livepeer.monster/hls/video+%s/index.m3u8"
+                      PLAYBACK_MANIFEST_URL="https://${{ inputs.region }}-staging-staging-catalyst-${{ inputs.pod-index }}.livepeer.monster/hls/video+%s/index.m3u8"
```

Comment:
I didn't test it but I guess it's a copy/paste mistake

Reply:
Good catch